app/grandchallenge/core/templates/base.html

```diff
@@ -66,11 +66,14 @@
         <div class="container pb-3 mt-3">
 
             {% block messages %}
+                {% include 'challenges/challenge_banner.html' %}
                 {% include "grandchallenge/partials/messages.html" %}
             {% endblock %}
 
             {% block outer_content %}
-                {% block topbar %}{% endblock %}
+                {% block topbar %}
+                    {% include 'challenges/challenge_topbar.html' with challenge=challenge %}
```

Comment:
`with challenge=challenge` should be unnecessary, right? Why is it needed here and not for the banner partial?

Reply:
oh yes, that can go indeed!